index.html

```diff
@@ -25,7 +25,7 @@
               <a href="javascript:void(0)" class="closebtn" onclick="closeNav()">&times;</a>
```

Comment:
- [ ] [OPTIONAL] Please consider modifying the design so it can be like the provided one